lib/lite_cable/server/client_socket/base.rb

```diff
@@ -142,10 +142,10 @@ def each_frame
           framebuffer = WebSocket::Frame::Incoming::Server.new(version: version)
 
           while IO.select([socket])
-            if socket.respond_to?(:recvfrom)
-              data, _addrinfo = socket.recvfrom(2000)
+            if socket.respond_to?(:recv)
+              data = socket.recv(2000)
```

Comment:
We don't need resolved address for TCP socket and https://github.com/oracle/truffleruby/issues/1550